BT-Tracking/Services/AppSettings.swift

```diff
@@ -14,9 +14,11 @@ struct AppSettings {
     private enum Keys: String {
         case appState
         case appFirstTimeLaunched
+
         case backgroundModeAlertShown
 
-        case lastProcessedFileName
+        case traveler
```

Comment:
Tim se mysli, ze uzivatel povolil spolupraci se zahranicim? Pokud ano, tak bych to prejmenoval na neco vystiznejsiho. Napr `efgsEnabled`.